src/modules/analytics/track.ts

```diff
@@ -77,6 +76,7 @@ import {
   ClaimNameSuccessAction
 } from 'modules/ens/actions'
 import { CREATE_COLLECTION_FORUM_POST_FAILURE, CREATE_COLLECTION_FORUM_POST_SUCCESS } from 'modules/forum/actions'
+import { getTransactionFromAction } from 'decentraland-dapps/dist/modules/transaction/utils'
```

Comment:
Could we move this to the top of the file! as per the [standards](https://github.com/decentraland/standards/blob/master/standards/react-redux.md#5-imports-order-on-files)

Reply:
Done